Gemfile.lock

```diff
@@ -1,15 +1,99 @@
 GEM
   remote: https://rubygems.org/
   specs:
+    addressable (2.7.0)
+      public_suffix (>= 2.0.2, < 5.0)
     ansi (1.5.0)
-    ast (2.4.0)
+    ast (2.4.2)
+    bcrypt (3.1.16)
+    bcrypt-ruby (3.1.5)
```

Comment:
can remove bcrypt-ruby dependency (according to error message when running bundle), and just keep bcrypt (as online above)